runtime/near-vm-runner/src/runner.rs

```diff
@@ -33,8 +33,8 @@ pub fn run<'a>(
     profile: &ProfileData,
 ) -> (Option<VMOutcome>, Option<VMError>) {
     run_vm(
-        code_hash,
-        code,
+        code.hash.as_ref().to_vec(),
+        code.code.as_slice(),
```

Comment:
I'll refactor the rest of vm-runner in a separate PR, after this interface PR is merged. That change would be large-ish, but it wont't touch runtime codewners.